CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ and this project adheres to
 ## Fixed
 
 🐛(frontend) invalidate queries after removing user #336
-
+🐛(backend) Docker compose up/down not working due to misconfigured depends_on at docker-compose.yml #339
```

Comment:
The changelog message is limited to 80 characters, we reference the PR id in the changelog.
```suggestion
🐛(backend) fix nginx docker container #340
```